components/application-mgt/org.wso2.carbon.device.application.mgt.publisher.ui/src/main/resources/publisher/src/components/Application/Edit/Base/baseLayout.css

```diff
@@ -60,8 +52,7 @@
 /*Tab styling*/
 div.tab {
     float: left;
-    border: 1px solid #ccc;
```

Comment:
The css classes files should go into the public directory right? Else we can't change the theming..